### DISCUSSION TITLE
paperspace / non google colab

### DISCUSSION THREAD

**Discussion by nawnie** [TheLastBen/fast-stable-diffusion#359] [Ideas]
I see you have added the drivers for the paperspace gfx cards, is there any plans to write a specific ipynb for paperspace, all the references to the drive seems to break it.

**Comment by TheLastBen** [comment]
I tried making a paperspace notebook, but the dependencies installation takes a longtime, so I put it low in the priority list

**Reply by nawnie** [reply]
fair enough.
